source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputChoiceSetRenderer.mm

```diff
@@ -38,11 +38,8 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
     std::shared_ptr<BaseCardElement> elem = [acoElem element];
     std::shared_ptr<ChoiceSetInput> choiceSet = std::dynamic_pointer_cast<ChoiceSetInput>(elem);
     // creates a tableview with pre-defined style
-    NSBundle *bundle = [NSBundle bundleWithIdentifier:@"MSFT.AdaptiveCards"];
-    if (!bundle) { // https://github.com/Microsoft/AdaptiveCards/issues/1834
-        return nil;
-    }
-    ACRInputTableView *choiceSetView = [bundle loadNibNamed:@"ACRInputTableView" owner:self options:nil][0];
+    //    ACRInputTableView *choiceSetView = [SWIFTPM_MODULE_BUNDLE loadNibNamed:@"ACRInputTableView" owner:self options:nil][0];
```

Comment:
do we need this comment? :)

Reply:
thanks, @paulcam206 I left a comment for removing this comment in my first pass-through, and forgot to submit my review of the first pass.